<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>1190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="onglet">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>1241</width>
      <height>1081</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tableau de Bord</string>
     </attribute>
     <widget class="QTableView" name="playerTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>821</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="trainingTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>630</y>
        <width>821</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="trainingButton">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>230</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Lancer la séance</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>400</y>
        <width>351</width>
        <height>169</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="signalGrid">
       <item row="0" column="0">
        <widget class="QLabel" name="greenCLabel">
         <property name="text">
          <string>green</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="redCLabel">
         <property name="text">
          <string>red</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Configuration</string>
     </attribute>
     <widget class="QLabel" name="playerNameLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Identification du joueur :</string>
      </property>
     </widget>
     <widget class="QLabel" name="removePlayerLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Supression du joueur :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rfidEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="delEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="showPlayerTable">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>901</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lname">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>350</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lnameEdit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>370</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="fnameEdit">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>370</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="addRFID">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>440</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="fname">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>350</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Prénom</string>
      </property>
     </widget>
     <widget class="QLabel" name="rfidlabel">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>420</y>
        <width>47</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TAG RFID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addPlayerB">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>480</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>1110</x>
        <y>360</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Test Laser</string>
      </property>
     </widget>
     <widget class="QPushButton" name="laserON">
      <property name="geometry">
       <rect>
        <x>1110</x>
        <y>390</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Laser On</string>
      </property>
     </widget>
     <widget class="QPushButton" name="laserOFF">
      <property name="geometry">
       <rect>
        <x>1110</x>
        <y>440</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>LaserOFF</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>310</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxPort">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>260</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="activePortButton">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>300</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Activer Port</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_readPeripherique">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>260</y>
        <width>201</width>
        <height>351</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Port">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>230</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="connectionLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1298</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>connexion database</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1278</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
